src/watchNPM.js

```diff
@@ -0,0 +1,29 @@
+const request = require('request-promise')
+const _ = require('lodash')
+const ChangesStream = require('changes-stream')
+const clean = require('normalize-registry-metadata')
+
+// will be structured similiarly to greenkeeper changes follow.js, except gutting the fluff
+
+// will hit up registry
+exports.checkWatcher = async function () { // TODO: fill the skeleton
+  request.get({
```

Comment:
I think you'll probably want to make use of `ChangesStream` here, similar to https://github.com/greenkeeperio/changes/blob/8ef0f9ba5659ae7eb6eafd9aacfc1ced18d4c773/lib/follow.js#L93